teams/2/informnational_pages.html

```diff
@@ -0,0 +1,458 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="assets/css/informnational_pages.css">
+    <title>&lt;title&gt; &lt;/title&gt; </title>
+</head>
+
+<body> &lt;body&gt; &lt;/body&gt;
+    <header> &lt;header&gt; &lt;/header&gt;
+        <nav>
+            <ul>&lt;ul&gt;
+                <img src="" alt=""> &lt;img/&gt;
+                <li><a href="page1.html">&lt;li&gt; &lt;a/&gt; &lt;li/&gt; </a></li>
+                <li><a href="page2.html">&lt;li&gt; &lt;a/&gt; &lt;li/&gt; </a></li>
+                <li><a href="page3.html">&lt;li&gt; &lt;a/&gt; &lt;li/&gt; </a></li>
+                <li><a href="page3.html">&lt;li&gt; &lt;a/&gt; &lt;li/&gt; </a></li>
+                <li><a href="page3.html">&lt;li&gt; &lt;a/&gt; &lt;li/&gt; </a></li>
+                <li><a href="page3.html">&lt;li&gt; &lt;a/&gt; &lt;li/&gt; </a></li>
+                &lt;/ul&gt;
+            </ul>
+            <ul>
+                <li>&lt;li&gt; &lt;li/&gt; </li>
+                <li>&lt;li&gt; &lt;li/&gt; </li>
+                <li>&lt;li&gt; &lt;li/&gt; </li>
+                <img src="" alt="<img/>">
+            </ul>
+        </nav>
+    </header>
+
+
+    <main> &lt;main&gt; &lt;/main&gt;
```

Comment:
This is not the main content. The table should be the main content

Reply:
Use the main tag for the main content of the page